taskcluster/docker/base/Dockerfile

```diff
@@ -35,7 +36,6 @@ RUN apt-get update -qq \
     # which we cannot navigate while building the Docker image.
     && apt-get install -y tzdata \
     && apt-get install -y openjdk-8-jdk \
-                          wget \
```

Comment:
`wget` is not used anymore thanks to this PR. Everything goes through `curl` which currently does the same job.